src/views/Invite/InviteEmail.js

```diff
@@ -142,15 +142,12 @@ export default function InviteEmail() {
   const isFailed = status === STATUS.FAILURE;
   const isDone = status === STATUS.SUCCESS;
 
-  const validateHasReceived = useCallback(
-    async email => {
-      const hasReceived = await getHasReceived(email);
-      if (hasReceived) {
-        return HAS_RECEIVED_TEXT;
-      }
-    },
-    [getHasReceived]
-  );
+  const validateHasReceived = useCallback(async email => {
+    // const hasReceived = await getHasReceived(email);
+    // if (hasReceived) {
+    // return HAS_RECEIVED_TEXT;
+    // }
+  }, []);
```

Comment:
Also what's this about? This same logic now lives in `Views/Invite/Email.js` it seems, so maybe a remnant of the move?

Reply:
reverted these changes, but maybe better to just turf the old views

Reply:
If they're unused they should just go.